src/ThreadsConfig.ts

```diff
@@ -13,14 +13,15 @@ export class ThreadsConfig extends Config {
     public token: string,
     public deviceId: string,
     public dev: boolean,
-    public apiScheme: string = 'https',
-    public api: string = 'cloud.textile.io',
-    public sessionPort: number = 8006,
+    public scheme: string = 'https',
+    public authApi: string = 'cloud.textile.io',
+    public authPort: number = 443,
+    public threadsApi: string = 'api.textile.io',
     public threadsPort: number = 6007,
     public transport: grpc.TransportFactory = grpc.WebsocketTransport(),
   ) {
     super()
-    this.host = `${this.apiScheme}://${this.api}:${this.threadsPort}`
+    this.host = `${this.scheme}://${this.threadsApi}:${this.threadsPort}`
```

Comment:
Might be nice to have a catch here as to whether port is undefined or not. Then only add the port if !== undefined?

Reply:
Something like:
```suggestion
    this.host = `${this.scheme}://${this.threadsApi}${this.threadsPort && ':' + this.threadsPort}`
```

Reply:
cool, we can do that as an enhancement if ever requested